### PR TITLE
Create jar.md

### PR DESCRIPTION
**Pull request by ArturSkowronski**
Adding information about handy usage of jar command

### CODE REVIEW

File: pages/common/jar.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# jar
+
+JAR (Java Archive) is a package file format used to distribute application software or libraries on the Java platform.
```
Comment (rubenvereecken) [inline pages/common/jar.md]: Should be preceded by a markdown blockquote symbol `>`.

**Comment by rubenvereecken** [comment]
Thanks for the page.

The linter failed, albeit extremely unelegantly. I'll add some tests for this kind of stuff.

**Comment by ArturSkowronski** [comment]
Thanks for info, I will try to run build locally

**Comment by rubenvereecken** [comment]
Just have a look at the inline comment and I'm happy :).

**Comment by ArturSkowronski** [comment]
I couldn't commit from local:
```pages/linux/mkfs.cramfs.md:1: TLDR013 Title should be alphanumeric with dashes, underscores or spaces
pages/linux/mkfs.exfat.md:1: TLDR013 Title should be alphanumeric with dashes, underscores or spaces
pages/linux/mkfs.fat.md:1: TLDR013 Title should be alphanumeric with dashes, underscores or spaces
pages/linux/mkfs.minix.md:1: TLDR013 Title should be alphanumeric with dashes, underscores or spaces
pages/linux/mkfs.ntfs.md:1: TLDR013 Title should be alphanumeric with dashes, underscores or spaces```
I got issues like that unfortunatelly. I was able to that from Github

**Comment by rubenvereecken** [comment]
I'm sorry for that @ArturSkowronski, these are all very recent changes. `tldr-lint` is ambitious but fidgety still. That's why Travis uses the master version whereas our build script uses the latest published version. I'll update the build script now.

**Comment by ArturSkowronski** [comment]
Ok, no problem, just wanted to inform you about this problem.

**Comment by rubenvereecken** [comment]
Sorry you had to go through that and thanks for the heads up. Thanks to you the next person not annotating their description nicely gets a pretty error. 

Your file is good syntax-wise, I'll let the others have a look at the actual contents. Thanks!

**Comment by ArturSkowronski** [comment]
Boy Scouts Rule. Happy to help :)

And thanks for great tool. Really love it.

**Comment by ostera** [comment]
Looks good to me! :+1:

File: pages/common/jar.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# jar
+
+> Tool empowering work with *.jar files, used to distribute application software or libraries on the Java platform.
+
+- Unzip *.jar/*.war file to the current directory:
```
Comment (rubenvereecken) [inline pages/common/jar.md]: I know this example was different in a previous commit of yours. Could you change this to reflect the changes to the command? Like "Extract all wars to the...", you know the lingo.

Comment (ArturSkowronski) [inline pages/common/jar.md]: Like now is better?

File: pages/common/jar.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# jar
+
+> Tool empowering work with *.jar files, used to distribute application software or libraries on the Java platform.
```
Comment (waldyrious) [inline pages/common/jar.md]: Maybe leave the asterisk out? Github seems to format the rest of the line as italics.

Also "Tool empowering work with ..." sounds a bit grandiose. Maybe change to something simpler/more direct?

**Comment by igorshubovych** [comment]
In the end, plz squash the commits.

**Comment by ArturSkowronski** [comment]
Commits squashed

**Comment by igorshubovych** [comment]
Thanks!

**Pull request merged**